src/bootstrap/src/ferrocene/code_coverage.rs

```diff
@@ -1,90 +1,73 @@
-use std::path::Path;
+use std::path::PathBuf;
 
-use crate::builder::{Builder, RunConfig, ShouldRun, Step};
-use crate::core::build_steps::tool::SourceType;
+use crate::BootstrapCommand;
+use crate::builder::Builder;
+use crate::core::build_steps::llvm::Llvm;
+use crate::core::builder::Cargo;
 use crate::core::config::TargetSelection;
-use crate::{BootstrapCommand, Kind, Mode};
 
-#[derive(Debug, Copy, Clone, PartialEq, Eq, Hash, PartialOrd, Ord)]
-pub(crate) struct ProfilerBuiltinsNoCore {
-    pub(crate) target: TargetSelection,
+pub(crate) fn instrument_coverage(cargo: &mut Cargo) {
```

Comment:
Does not have public does it?

```suggestion
fn instrument_coverage(cargo: &mut Cargo) {
```

Reply:
We could also get rid of the function at all, since it is just one line and only called once.

Reply:
The function is also called from `src/core/build_steps/compile.rs` to instrument coverage for libcore itself. I'd still like to keep it around if we need to do extra code to instrument a piece of code, so that we can change it in only one place.